src/lte_link.rs

```diff
@@ -133,7 +133,7 @@ impl Drop for LteLink {
         if ACTIVE_LINKS.fetch_sub(1, Ordering::SeqCst) == 1 {
             // Turn off the network side of the modem
             // We need to send this blocking because we don't have async drop yet
-            crate::at::send_at_blocking::<0>("AT+CFUN=20").unwrap();
+            crate::at::send_at_blocking::<8>("AT+CFUN=20").unwrap();
```

Comment:
This shouldn't be necessary... A buffer of 0 should just have nothing written to it. So the bug is not that this was set to zero, the bug is that it crashes because of the 0

Reply:
It crashes when I set it to 4 too.

Reply:
ok, I see. do you want a check then to see if it's 0 in the send_at function?

Reply:
Hmmmm, I see what is happening. Apparently with the blocking function you get an error if the buffer you give it is too small. I don't know what the correct thing to do here is.

Perhaps the best thing is to detect when the capacity is 0 and then use the printf function instead of the command function.
Printf only returns if there was an error or not and no response.

I'll try that out now